e2e/tests/interchain_accounts/params_test.go

```diff
@@ -0,0 +1,91 @@
+package interchain_accounts
+
+import (
+	"context"
+	"testing"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+
+	controllertypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
+	icatypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/stretchr/testify/suite"
```

Comment:
a bit unconventional import structing. Usually its 
```
// standard lib

// external imports

// ibc-go imports
```

but we haven't established a practice for individual go.mod's still within the ibc-go repo

Reply:
yeah I'll fix this up. Ideally we should add the same linting targets in the e2e makefile to have all go code follow a similar structure.